src/pages/SearchPage.js

```diff
@@ -72,40 +76,15 @@ export default function SearchPage({ year, semester }) {
                       <span className="schoolName">
                         {schools.data[schoolCode]
                           ? schools.data[schoolCode].name
-                          : ""}
+                          : findSchool(schoolCode)}
                       </span>
                     </div>
                   </Link>
-                  {Object.keys(departments.data[schoolCode])
```

Comment:
Removing the schools makes the bottom feel a bit empty. Maybe we should talk with Ethan to see what would be a good fix for this. Should be fine for now though.

Reply:
@EthanPrintz

Reply:
Thanks for the ping– I think if we divide it into two separate columns with headings of Undergraduate/Graduate (manually tagged I guess?) and move name-less schools to a single line at the bottom it'll help to increase visual density as well as improving parsing at a glance.

Reply:
I have changed this to 2 column with undergraduate and graduate. Lmk!

Reply:
Awesome! Looks great, I think that's helped quite a bit. Once we merge this back to the main branch I'll play around with some small tweaks to further increase visual density, but I think this is definitely a good place to have it.